mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule Mixduty.MixProject do
   def project do
     [
       app: :mixduty,
-      version: "0.1.1",
+      version: "1.0.0",
```

Comment:
I'd make it `0.2.0`.

Reply:
Why do you think that it is a minor changes?